newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -333,11 +361,11 @@ export const LeaderboardAdmin = ({
   };
 
   const onResetLeaderboard = async (i18n: I18nType) => {
-    const answer = Window.showConfirmDialog(
-      i18n._(
-        t`All current entries will be deleted, are you sure you want to reset this leaderboard? This can't be undone.`
-      )
-    );
+    if (!currentLeaderboard) return;
+    const answer = showConfirmation({
```

Comment:
by the way, should this be a "deleteConfirmation" dialog? rather than showConfirmation?

Reply:
I'm not sure and since we cannot change the button (of the dialog) from Delete to Reset/Confirm/Continue, I would say let's try it with the confirm dialog.